gatsby-browser.js

```diff
@@ -177,18 +177,18 @@ export const onClientEntry = () => {
 }
 
 const scrollToAnchor = (location) => {
-    // Check for location so build does not fail
-    if (location && location.hash) {
-        window.scrollTo({
-            top: window.innerHeight,
-            behavior: 'smooth',
-        })
+    if(location.hash){
+        const elem = document.querySelector(location.hash);
+        if(elem){
+            elem?.scrollIntoView({ behavior: "smooth" })
```

Comment:
hi bro since we are already checking elem by if condition , can we remove **?** here ?
and otherwise 
we can elem?.scrollIntoView({ behavior: "smooth" }) ||  window.scrollTo(0, 0) use this without  if(elem)

Reply:
Thanks.